VideoLocker/src/main/java/org/edx/mobile/module/notification/EdxParsePushBroadcastReceiver.java

```diff
@@ -32,37 +32,23 @@
 public class EdxParsePushBroadcastReceiver extends ParsePushBroadcastReceiver {
     protected final Logger logger = new Logger(getClass().getName());
 
+    private boolean enableFlyinWindow = false;
```

Comment:
@aleffert yes. as it is a temporary flag,  it may not be a good idea to put it in the configuration file.